attestation-service/attestation-service/src/bin/restful-as.rs

```diff
@@ -95,7 +95,12 @@ async fn main() -> Result<(), RestfulError> {
     let server = HttpServer::new(move || {
         App::new()
             .service(web::resource(WebApi::Attestation.as_ref()).route(web::post().to(attestation)))
-            .service(web::resource(WebApi::Policy.as_ref()).route(web::post().to(set_policy)))
+            .service(
+                web::resource(WebApi::Policy.as_ref())
+                    .route(web::post().to(set_policy))
+                    .route(web::get().to(list_policies))
```

Comment:
Do we need an API to list concrete policy content? like
GET `/policy` would return policy list
GET  `/policy/<id>` would return content of policy <id>

Reply:
At present, one GET endpoint to get all polices can already meet the requirements. I think the new endpoint you mentioned can be implemented when further requirements arise in the future.

Reply:
Good idea, I'll implement this.